requirements-dev.in

```
@@ -4,4 +4,4 @@
 
 # additional requirements can be listed here
 testcontainers>=3.4.1
-hexkit[s3]>=0.10,<0.11
+hexkit[s3]>=0.11,<0.12
```

Shouldn't that correspond to what's in the pyproject.toml?
Don't know if the same notation is supported, but the lower bound should be 0.11.1

---

Dev dependencies don't need to have an upper bound. So can be:
```suggestion
hexkit[s3]>=0.11
```